runtime/src/lib.rs

```diff
@@ -206,7 +229,19 @@ impl<'a> Drivers<'a> {
     }
 }
 
-fn wait_for_cmd(_mbox: &mut Mailbox) {
+/// Run pending jobs and enter low power mode.
+fn goto_idle(drivers: &mut Drivers) {
```

Comment:
nit: `go_idle`? I think the name `goto_idle` could be misinterpreted as performing a goto jump.

Reply:
Done.